docs/sources/basics/exemplars/_index.md

```diff
@@ -21,4 +21,6 @@ Grafana shows exemplars alongside a metric in the Explore view and in dashboards
 
 {{< figure src="/static/img/docs/v74/exemplars.png" class="docs-image--no-shadow" max-width= "750px" caption="Screenshot showing the detail window of an Exemplar" >}}
 
+> **Note:** If there are multiple exemplars for the same metric name, Grafana shows only the first.
```

Comment:
This is not true. When there are multiple queries with the same metric name but with different percentile for example `histogram_quantile(0.95, sum(rate(tns_request_duration_seconds_bucket[5m])) by (le))` and `histogram_quantile(0.50, sum(rate(tns_request_duration_seconds_bucket[5m])) by (le))` only one exemplar request is going to fire and you will only see one kind of symbol because exemplars are associated to labels. @mdisibio please correct me if I'm wrong.